snippets/juxt.md

```diff
@@ -0,0 +1,28 @@
+---
+title: juxt
+tags: array,function,advanced
+---
+
+Takes several functions as argument and returns a fn that is the juxtaposition of those fns.
+
+juxt(fn1, fn2, fn3, ...) use `Array.prototype.map()` to return a `func` that can take a variable number of `args`.
+When `func` is called, return a array containing the result of applying each fn (fn1, fn2, fn3, ...) to the `args`.
+
+```js
+const juxt = (...fns) => (...x) => [...fns].map(fn => [...x].map(x => fn(x)));
```

Comment:
```suggestion
const juxt = (...fns) => (...args) => [...fns].map(fn => [...args].map(args => fn(args)));
```

Couldn't the last `.map` be simplified to `.map(fn)` with `args` being implicit? Could you check that?

Reply:
Yeap! That's right, can be simplified.